app/controllers/posts_controller.rb

```diff
@@ -6,4 +6,27 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: @post } }
+    end
+  end
+
+  def create
+    post = Post.new(params.require(:post).permit(:Title, :Text)
+    .merge(author: current_user, CommentsCounter: 0, LikesCounter: 0))
```

Comment:
- [ Optional] Please it would be better to put the  params inside a method instead of getting the parameters from the params to object to encapsulate the permissible parameters 
you can read more about it in this lesson Lesson link about the strong [parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).